androidClient/app/src/main/java/com/csatimes/dojma/campuswatch/ShortsActivity.java

```diff
@@ -1,34 +1,44 @@
 package com.csatimes.dojma.campuswatch;
 
 import android.os.Bundle;
+import android.view.Window;
+import android.widget.Toolbar;
 
 import com.csatimes.dojma.R;
-import com.csatimes.dojma.models.ShortsItem;
+import com.google.android.material.appbar.AppBarLayout;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
 import androidx.viewpager.widget.ViewPager;
-import io.realm.Realm;
 
-public class ShortsActivity extends AppCompatActivity {
+public class ShortsActivity extends AppCompatActivity implements OnShortClicked {
+    private AppBarLayout appBarLayout;
+    private boolean clicked;
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_shorts);
-        ((ViewPager) findViewById(R.id.vp_shorts)).setAdapter(new ShortsAdapter());
-        findViewById(R.id.fab_back).setOnClickListener(view -> onBackPressed());
-        findViewById(R.id.fab_done).setOnClickListener(view -> {
-            final Realm db = Realm.getDefaultInstance();
-            db.executeTransaction(realm -> {
-                for (final ShortsItem item : realm.where(ShortsItem.class).findAll()) {
-                    final ShortsItem updatedData = realm.copyFromRealm(item);
-                    updatedData.isRead = true;
-                    realm.insertOrUpdate(updatedData);
-                }
-            });
-            db.close();
-            onBackPressed();
-            finish();
-        });
+        final Window window = getWindow();
+        window.setStatusBarColor(ContextCompat.getColor(this, R.color.black));
+        window.setNavigationBarColor(ContextCompat.getColor(this, R.color.black));
+        appBarLayout = findViewById(R.id.shorts_appbar);
+        clicked = true;
+        appBarLayout.setExpanded(clicked);
+        Toolbar toolbar = findViewById(R.id.shorts_toolbar);
+        toolbar.setNavigationIcon(R.drawable.ic_arrow_back);
+        toolbar.setTitle(R.string.campus_watch);
+        toolbar.setNavigationOnClickListener(v -> onBackPressed());
+        ViewPager viewPager = findViewById(R.id.vp_shorts);
+        ShortsAdapter shortsAdapter = new ShortsAdapter();
```

Comment:
pass the onShortClicked in constructor

Reply:
I am currently using the setter approach, I believe that is considered better for quality of code.